tests/TestCase/I18n/DateTimeTest.php

```diff
@@ -412,6 +412,12 @@ public function testI18nFormat(): void
         $result = $time->i18nFormat(IntlDateFormatter::FULL, 'Asia/Tokyo', 'ja-JP@calendar=japanese');
         $expected = '平成22年1月14日木曜日 22時59分28秒 日本標準時';
         $this->assertTimeFormat($expected, $result);
+
+        // Test with milliseconds
+        $timeMillis = new FrozenTime('2014-07-06T13:09:01.523000+00:00');
```

Comment:
```suggestion
        $timeMillis = new DateTime('2014-07-06T13:09:01.523000+00:00');
```

5.x doesn't have `FrozenTime`, `DateTime` is the immutable class.